apps/workspaces/enums.py

```
@@ -0,0 +1,8 @@
+from fyle_accounting_library.enums import CacheKeyEnum
+
+
+class CacheKeyEnum(CacheKeyEnum):
```

Pylint (3.3.8)

[error] class already defined line 1

(E0102)